packages/cdai/.stylelintrc.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
```

Comment:
Is there a script in `cdai/package.json` to run this or will it be covered by `lint:style` in the project's root?

Reply:
It is covered by the root. If you look at the ci-check it shows what would be errors outside of CDAI as warnings

![image](https://user-images.githubusercontent.com/15086604/95310233-11aa2280-0884-11eb-8f85-6356a644ecd7.png)